benchmark.py

```diff
@@ -95,7 +95,8 @@ def _get_commits_topological(
 
     if len(sorted_commit_list) != len(commits_sha_set):
       raise ValueError(
-          "The following commits weren't found in the repo in branch master: %s"
+          "The following commits weren't found in the repo in branch master: %s." \
+          "Please use the full SHA digest."
```

Comment:
really? why?

Reply:
GitPython Commit doesn't directly support short SHA and we'll have to add some custom logic to address this. I'll do that in a separate follow-up PR right after this.